routes/auth-routes.js

```diff
@@ -15,18 +21,18 @@ router.post("/signin", async (req, res) => {
     ]);
 
     if (!result.rows.length) {
-      return res.status(400).json({ 
-        status: 'error',
-        error: "Invalid email" });
+      return res
+        .status(STATUSCODE.BAD_REQUEST)
+        .json(errorResponse(STATUS.Error, "Invalid email"));
     }
 
     const user = result.rows[0];
     const isPasswordValid = await bcrypt.compare(password, user.password);
 
     if (!isPasswordValid) {
-      return res.status(400).json({ 
-        status: 'error',
-        error: "Invalid password" });
+      return res
+        .status(STATUSCODE.BAD_REQUEST)
+        .json(errorResponse(STATUS.Error, "Invalid Password"));
```

Comment:
We Shouldnt Be Specific For Each Email And Password
We Should Just Tell The User: **Invalid Email Or Password**